### DISCUSSION TITLE
How to create an inbox for Whatsapp using the API?

### DISCUSSION THREAD

**Discussion by lukibsubekti** [chatwoot/chatwoot#5888] [Chatwoot APIs] [answered]
I've checked the documentation [here](https://www.chatwoot.com/developers/api/#tag/Inboxes/operation/inboxCreation). But, it shows only an example of `web_widget` creation. How to create an inbox for Whatsapp? What are the type and the payload? Is there any latest documentation that has been released?
Thank you, guys.

**Comment by sojan-official** [comment, marked as answer]
@lukibsubekti  we don't have documentation for this as its not a common use case.  but you can change the payload of `channel` attribute to the appropriate values for WhatsApp inbox.  

example: 

```
{"name":"test","channel":{"type":"whatsapp","phone_number":"+2343243","provider":"whatsapp_cloud","provider_config":{"api_key":"fadsfsfdsafas","phone_number_id":"23423432","business_account_id":"242343243242"}}}
```
